sklearn/utils/estimator_checks.py

```diff
@@ -67,7 +67,14 @@
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
 
 
-def _yield_non_meta_checks(name, Estimator):
+def assert_almost_equal_dense_sparse(x, y, err_msg=''):
+    if sparse.issparse(x):
+        assert_allclose(x.data, y.data, err_msg=err_msg)
```

Comment:
So maybe I should not have moved it and not promised that it worked in all cases?